l10n_ch_account_tags/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+The migration of this module from 13.0 to 14.0 was financially supported by Camptocamp
```

Comment:
I don't think this should be in the `DECRIPTION.rst` file.
Maybe `CREDITS.rst` ?

Reply:
I got it, thanks you !